client/dashboard/data/index.ts

```diff
@@ -99,34 +99,28 @@ export const fetchSiteMediaStorage = async ( id: string ): Promise< MediaStorage
 };
 
 export const fetchSiteMonitorUptime = async (
-	site: Site
+	id: string
 ): Promise< MonitorUptime | undefined > => {
-	if ( ! site ) {
-		return Promise.reject( new Error( 'Site is undefined' ) );
-	}
-	if ( ! site.jetpack || ! site.jetpack_modules?.includes( 'monitor' ) ) {
-		return;
+	if ( ! id ) {
+		return Promise.reject( new Error( 'Site ID is undefined' ) );
 	}
 	return wpcom.req.get(
 		{
-			path: `/sites/${ site.ID }/jetpack-monitor-uptime`,
+			path: `/sites/${ id }/jetpack-monitor-uptime`,
 			apiNamespace: 'wpcom/v2',
 		},
 		{ period: '30 days' }
 	);
 };
 
-export const fetchPHPVersion = async ( site: Site ): Promise< string | undefined > => {
-	if ( ! site ) {
-		return Promise.reject( new Error( 'Site is undefined' ) );
-	}
-	if ( ! site.options.is_wpcom_atomic ) {
-		return;
+export const fetchPHPVersion = async ( id: string ): Promise< string | undefined > => {
+	if ( ! id ) {
+		return Promise.reject( new Error( 'Site ID is undefined' ) );
```

Comment:
Why do we have these tests, feels like they're redundant as the typescript type says the id is mandatory. (same in the other functions)

Reply:
Maybe we could use the `enabled` prop for these.

Reply:
Yes, I don't know why they were added. I can remove them

Reply:
We don't need enabled, they're not hooks so it can be conditional if needed